soc/ti/mspm0/mspm0g3xxx/soc.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2024 Texas Instruments
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <ti/driverlib/m0p/dl_core.h>
+#include <soc.h>
+
+/*
+ *  ======== SYSCFG_DL_init ========
+ *  Perform any initialization needed before using any board APIs
+ */
```

Comment:
do you need these commends? the first line does not add anything, just drop it, the other one up to you but I'd drop it as well, the function name seems descriptive enough

Reply:
I'll clean up this file actually